<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manBooks</class>
 <widget class="QDialog" name="manBooks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="addBookButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Book</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteBookButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>340</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Book</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>521</width>
     <height>192</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>3</number>
   </property>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Title</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Author</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Genre</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Available</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>001</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>The Great Gatsby</string>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>F. Scott Fitzgerald</string>
    </property>
   </item>
   <item row="0" column="3">
    <property name="text">
     <string>Fiction</string>
    </property>
   </item>
   <item row="0" column="4">
    <property name="text">
     <string>Yes</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>002</string>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string>1984</string>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string>George Orwell</string>
    </property>
   </item>
   <item row="1" column="3">
    <property name="text">
     <string>Fiction</string>
    </property>
   </item>
   <item row="1" column="4">
    <property name="text">
     <string>Yes</string>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string>003</string>
    </property>
   </item>
   <item row="2" column="1">
    <property name="text">
     <string>Brief Answers to the Big Questions</string>
    </property>
   </item>
   <item row="2" column="2">
    <property name="text">
     <string>Stephen Hawking</string>
    </property>
   </item>
   <item row="2" column="3">
    <property name="text">
     <string>Science</string>
    </property>
   </item>
   <item row="2" column="4">
    <property name="text">
     <string>No</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>641</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Book Management</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
